### PR TITLE
Display cards in game

### PR DESCRIPTION
**Pull request by glitchwolfe**

### CODE REVIEW

File: Assets/Scripts/Cards/DisplayCard.cs [hunk 1]
```diff
@@ -2,30 +2,33 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
+using TMPro;
 
 public class DisplayCard : MonoBehaviour {
```
Comment (AesteroidBlues) [inline Assets/Scripts/Cards/DisplayCard.cs]: Could we call this class something like "CardView" or "CardViewController" instead? In Visual Studio you can refactor a class name by pressing F2 when you've got it selected

File: Assets/Scripts/Cards/DisplayCard.cs [hunk 2]
```diff
@@ -2,30 +2,33 @@
...
+	public TextMeshProUGUI attack;
+	public TextMeshProUGUI health;
 
 	public Image cardArt;
 
 	// Use this for initialization
 	void Start () {
-		name.text = card.name;
-		description.text = card.description;
+		cardName.text  = card.cardName;
```
Comment (AesteroidBlues) [inline Assets/Scripts/Cards/DisplayCard.cs]: These tabs are really starting to get to me, we should have a conversation in Discord about getting everyone on tabs or spaces and not both 😄

File: Assets/Scripts/Cards/Card.cs [hunk 3]
```diff
@@ -3,23 +3,23 @@
...
 
-public class Card : ScriptableObject {
+public abstract class Card : ScriptableObject {
 
 	public Sprite cardArt;
 
 	public string cardName;
 	public string type;
-	public string description;
+	public string abilities;
```
Comment (AesteroidBlues) [inline Assets/Scripts/Cards/Card.cs]: I wonder if this should be a `List<string>` instead so we can have each ability stored in a separate string? We should talk to Karen and Dexter about how they'd prefer to keep abilities (or even if cards will have more than one?)

**Review by AesteroidBlues — APPROVED** [review]
Some super minor nitpicks but looks good

File: Assets/Scripts/Cards/DisplayCard.cs (hunk 2 above)
Comment (glitchwolfe) [reply Assets/Scripts/Cards/DisplayCard.cs]: Hah that drives me crazy too (I use 2 char tabs locally). Is that something we can settle with a .gitattributes file?

**Pull request merged**